app/models/shipit/task.rb

```diff
@@ -429,5 +434,11 @@ def duplicate_task
       copy_task.ended_at = nil
       copy_task
     end
+
+    def append_retry_if_necessary(event)
+      return event unless retries_configured?
+      return "#{event}_retry".to_sym if (status == "running" && retry_attempt > 0) || status == "timedout"
```

Comment:
A bit pedantic/YAGNI/futureproofing: it might be a bit more correct if the status check here was `RETRIABLE_STATUSES.include?(status) || (status == "running" ...)` but that's not as important until we have more than one retriable status.

Reply:
Logic has been removed as per https://github.com/Shopify/shipit-engine/pull/1111#discussion_r510295799